File: Erlang/output.erl
-module(output).
-export([out/4, out/5]).

out(Name, Pi, N, ElapseTime) ->
    io:format("==================== ~s~n", [Name]),
    io:format("    π = ~p~n", [Pi]),
    io:format("    iteration count = ~p~n", [N]),
    io:format("    elapse time = ~p~n", [ElapseTime]).

out(Name, Pi, N, ElapseTime, NWorkers) ->
    out(Name ++ ": task count " ++ integer_to_list(NWorkers), Pi, N, ElapseTime),
    io:format("    scheduler count = ~p~n", [erlang:system_info(schedulers)]).

File: C#/Output.cs
/*
 *  Output routines for calculation of π using quadrature.
 *
 *  Copyright © 2015  Russel Winder
 */

public class Output {

  public static void output(string name, double pi, long iterations, double elapseTime, string additional = "") {
    System.Console.WriteLine("========================== " + name + "\n" +
                             "\tπ = " + pi + "\n" +
                             "\titerations = " + iterations + "\n" +
                             "\telapse time = " + elapseTime +
                             additional);
  }

  public static void output(string name, double pi, long iterations, double elapseTime, int threadCount) {
    output(
           name + " (thread count = " + threadCount + ")",
           pi,
           iterations,
           elapseTime,
           "\n\tnumber of processors = " + System.Environment.ProcessorCount);
  }

}
